scripts/safe_rename.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs')
+const fetch = require('node-fetch')
+
+let DEBUG = false
+
+const log = (...args) => {
+    if (DEBUG) {
+        console.log.apply(null, args)
+    }
+}
+
+const formatDate = (date) => {
```

Comment:
Nit: If the brackets contain only a return statement, you can drop the brackets and the return keyword.

Reply:
@samwinslow want to acknowledge this but I'm not going to update this time around as I want to get it merged and I think there are a number of functions that I should update if I were to take this on board.